src/Apps/Order/Components/OfferInput.tsx

```diff
@@ -0,0 +1,42 @@
+import Input from "Components/Input"
+import React from "react"
+
+export interface OfferInputProps {
+  id: string
+  showError?: boolean
+  onChange: (value: number) => void
+}
+
+export class OfferInput extends React.Component<OfferInputProps> {
```

Comment:
I could see this (in the future) becoming a _type_ of input that we could reuse and inject with a unique error message. But this seems like a fine abstraction for now 👍